can't build

### Priority

Missing Content

### Area

- [ ] Datapack
- [x] Source
- [ ] Map
- [ ] Other

### What is missing?

/usr/include/c++/11/bits/unordered_map.h:105:18:   required from ‘class std::unordered_map<std::__cxx11::basic_string<char>, WheelSpells::Bonus>’
/home/canary/src/creatures/players/components/wheel/player_wheel.hpp:535:54:   required from here
/usr/include/c++/11/bits/stl_pair.h:218:11: error: ‘std::pair<_T1, _T2>::second’ has incomplete type
  218 |       _T2 second;                ///< The second member
      |           ^~~~~~
In file included from /home/canary/src/creatures/players/player.hpp:26,
                 from /home/canary/src/server/network/protocol/protocolgame.cpp:29,
                 from /home/canary/build/linux-release/src/CMakeFiles/canary_lib.dir/Unity/unity_21_cxx.cxx:7:
/home/canary/src/creatures/players/components/wheel/player_wheel.hpp:38:16: note: forward declaration of ‘struct WheelSpells::Bonus’
   38 |         struct Bonus;
      |                ^~~~~
ninja: build stopped: subcommand failed.

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

--- a/src/creatures/players/components/wheel/player_wheel.hpp
+++ b/src/creatures/players/components/wheel/player_wheel.hpp
@@ -11,6 +11,7 @@
 
 #include "creatures/creatures_definitions.hpp"
 #include "creatures/players/components/wheel/wheel_definitions.hpp"
+#include "creatures/players/components/wheel/wheel_spells.hpp"
 
 class Creature;
 class IOWheel;

